### PR TITLE
Set up for deployment

### PR DESCRIPTION
**Pull request by hexxu**
Set up environment variables for deployment.

### CODE REVIEW

File: client/src/redux/workersRedux/workersService.js [hunk 1]
```diff
@@ -54,15 +56,15 @@ const updateWorker = async ({ id, name, email, phone, address, hRate, trades, pC
 };
 
 const getWorker = async (id) => {
-  const response = await fetch(`http://localhost:3001/workers/${id.id}`, {
+  const response = await fetch(`${SERVER_BASE_URL}/workers/${id.id}`, {
```
Comment (ryanzhxu) [inline client/src/redux/workersRedux/workersService.js:59]: should this be `${id}`?

Comment (hexxu) [reply client/src/redux/workersRedux/workersService.js:59]: Not sure. Maybe @zczhou16 can answer?

Comment (zczhou16) [reply client/src/redux/workersRedux/workersService.js:59]: Will fix that after this weekend.

Comment (ryanzhxu) [reply client/src/redux/workersRedux/workersService.js:59]: 🙏

**Review by ryanzhxu — APPROVED** [review]

**Pull request merged**